docs/src/pages/demos/steppers/steppers.md

```diff
@@ -64,11 +64,10 @@ Labels can be placed below the step icon by setting the `alternativeLabel` prope
 
 {{"demo": "pages/demos/steppers/VerticalLinearStepper.js"}}
 
-## Customized Steppers
+## Customized Stepper
 
-If you have read the [overrides documentation page](/customization/overrides/)
-but you are not confident jumping in,
-here are examples of how you can change the look of a stepper.
+Here is an example of customizing the component. You can learn more about this in the
```

Comment:
What do you think of using the plural? It's guarantee that we will add more demos and that we will forget to update it. I think that we can start this the plural and forgets about it, we will eventually have multiple demos for all the customizations.

Reply:
I understand your point, but I think we will catch it. I thought about the translation impact, but TM will take care of any future change from singular to plural.